docker-compose-for-local.yaml

```diff
@@ -29,7 +29,7 @@ services:
   nginx:
     build:
       context: biap-app-ui-front
-      dockerfile: Dockerfile
+      dockerfile: DockerfileWithoutSSL
```

Comment:
is dockerfilewithoutssl updated

Reply:
in here 
https://github.com/ONDC-Official/ondc-sdk/blob/d4bea5b6f37405956173c48f899ce20e637ef717/deploying_ansible/buyer-app-run.yaml#L74C5-L81C1

we are using normal dockerfile without ssl in ansible

Reply:
understood, this was throwing error on local-setup, for now reverting this Dockerfile change, will fix with ansible together